### DISCUSSION TITLE
font

### DISCUSSION THREAD

**Discussion by wen0271** [Team3-repo/Camp_Win_Oh#2] [General]
# 中文字體

```js
@import url('https://fonts.googleapis.com/css2?family=Zen+Maru+Gothic:wght@400;700&display=swap');
```
font-family: "Zen Maru Gothic", sans-serif;

# 英文字體

```js
@import url('https://fonts.googleapis.com/css2?family=Poetsen+One&display=swap');
```

font-family: 'Poetsen One', sans-serif;

**Comment by Tseng24** [comment]
![CampWinOh_logo](https://github.com/user-attachments/assets/c4554260-a73a-444e-b0d4-b5333928de2d)
